shared/reducers/tracker.js

```diff
@@ -457,6 +435,14 @@ function remoteProofToProof (rp: RemoteProof, lcr: ?LinkCheckResult): Proof {
     statusMeta = proofStatusToSimpleProofMeta(lcr.proofResult.status)
   }
 
+  const isTracked = !!(lcr && lcr.diff && lcr.diff.type === identify.TrackDiffType.none && !lcr.breaksTracking)
+
+  // whatevz, tracked a broken thing
+  if (lcr && lcr.proofResult && lcr.proofResult.status !== identify.ProofStatus.ok && isTracked) {
+    diffMeta = metaIgnored
+    statusMeta = null
+  }
```

Comment:
We already have code that maps between `lcr.proofResult.status` and `lcr.diff.type`. We should update those function (`proofStatusToSimpleProofMeta` and `trackDiffToSimpleProofMeta`) to return something for `TrackDiffType.none` and `ProofStatus.ok` instead of duplicating this logic here.